files/assemble.py

```diff
@@ -65,11 +65,11 @@
     default: null
   remote_src:
     description:
-      - If False, it will search for src at originating/master machine, if True it will
-        go to the remote/target machine for the src. Default is True.
-    choices: [ "True", "False" ]
+      - If C(no), it will search for src at originating/master machine, if C(yes) it will
+        go to the remote/target machine for the src. Default is C(yes).
+    choices: [ "yes", "no" ]
     required: false
-    default: "True"
+    default: "no"
```

Comment:
You are mistaken; the default is 'no'.

The implementation (lines 161 onward) states:

```
     module = AnsibleModule(
         # not checking because of daisy chain to file module
         argument_spec = dict(
             src = dict(required=True),
             delimiter = dict(required=False),
             dest = dict(required=True),
             backup=dict(default=False, type='bool'),
             remote_src=dict(default=False, type='bool'),
             regexp = dict(required=False),
             ignore_hidden = dict(default=False, type='bool'),
             validate = dict(required=False, type='str'),
         ),
         add_file_common_args=True
     )
```

The default value for remote_src is 'no'. This change corrects the documentation to match the implementation.